doc/examples/with_widgets_overlay/lib/main.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:flame/game.dart';
+import 'package:flame/gestures.dart';
+
+class ExampleGame extends Game with HasWidgetsOverlay, TapDetector {
+  bool isPaused = false;
+
+  @override
+  void update(double dt) {}
+
+  @override
+  void render(Canvas canvas) {
+    canvas.drawRect(const Rect.fromLTWH(100, 100, 100, 100),
+        Paint()..color = const Color(0xFFFFFFFF));
+  }
+
+  @override
+  void onTap() {
+    if (isPaused) {
+      removeWidgetOverlay("PauseMenu");
```

Comment:
use single quotes instead (everywhere)

Reply:
we should have a linter for this kind of thing, as it is really easy to forget/get confused